<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SmartWay</class>
 <widget class="QMainWindow" name="SmartWay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shortcut</string>
  </property>
  <property name="windowIcon">
   <iconset resource="smartway.qrc">
    <normaloff>:/imagens/shortcut.png</normaloff>:/imagens/shortcut.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="frameConfig">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>18</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QGroupBox" name="groupMapaArea">
         <property name="title">
          <string>Definição de Área</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="lblAltura">
            <property name="text">
             <string>Altura:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtAltura">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblLargura">
            <property name="text">
             <string>Largura:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txtLargura">
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDefinirMapa">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Definir Área</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioPartida">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Partida</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioChegada">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Chegada</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioObstaculos">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Obstáculos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupPesos">
         <property name="title">
          <string>Definição Pesos</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="lblHorizontal">
            <property name="text">
             <string>Horizontal:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="txtPesoHorizontal">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblVertical">
            <property name="text">
             <string>Vertical:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="txtPesoVertical">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lblDiagonal">
            <property name="text">
             <string>Diagonal:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="txtPesoDiagonal">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkPeso">
         <property name="text">
          <string>Mostrar pesos</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCalcular">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Calcular Rota</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="tabelaMapa">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="iconSize">
       <size>
        <width>41</width>
        <height>41</height>
       </size>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>48</number>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>48</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>48</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>48</number>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>txtAltura</tabstop>
  <tabstop>txtLargura</tabstop>
  <tabstop>btnDefinirMapa</tabstop>
  <tabstop>radioPartida</tabstop>
  <tabstop>radioChegada</tabstop>
  <tabstop>radioObstaculos</tabstop>
  <tabstop>txtPesoHorizontal</tabstop>
  <tabstop>txtPesoVertical</tabstop>
  <tabstop>txtPesoDiagonal</tabstop>
  <tabstop>chkPeso</tabstop>
  <tabstop>btnCalcular</tabstop>
 </tabstops>
 <resources>
  <include location="smartway.qrc"/>
 </resources>
 <connections/>
</ui>
